### DISCUSSION TITLE
How to make a crosshair "aim" in third person correctly?

### DISCUSSION THREAD

**Discussion by vitalspace** [enable3d/enable3d#314] [Help / Questions]
Hi, I'm using "Third Person Controls" to do an "aim" crosshair, in third person. but I have no idea how to set it right.

i have the follow

```
            this.controls = new ThirdPersonControls(this.camera, this.man, {
              offset: new THREE.Vector3(0.7, 1, 0),
              targetRadius: 3,
            })
```

and the result: 

https://user-images.githubusercontent.com/29004070/184522069-898de100-d33a-4d2d-95d7-820c35075797.mp4

i want something like 

https://youtu.be/FbM4CkqtOuA

**Comment by yandeu** [comment]
This is just a 2d image rendered in top of the 3d scene.
